rpt/operationslist_fr.properties

```diff
@@ -0,0 +1,7 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+Operation.Type  = Typologie / Maladie

```

Comment:
There's no need to add new keys in the language other than "en". Please revert this change.